### DISCUSSION TITLE
make setup error report

### DISCUSSION THREAD

**Discussion by nicedayd** [efabless/caravel_user_project#268] [Q&A]
Hello,
Could somebody support to fix following issue?
Background: run the [caravel_user_project](https://github.com/efabless/caravel_user_project/) setup in my laptop in Win10 with make command; installed MinGW to support "make" in win10. 
Problem: when run the make setup, it reported following error
---------
FIND: Parameter format not correct
FIND: Parameter format not correct
FIND: Parameter format not correct
make -f /caravel/Makefile setup
make[1]: Entering directory 'C:/GitHub Study/caravel_user_project'
make[1]: /caravel/Makefile: No such file or directory
make[1]: *** No rule to make target '/caravel/Makefile'.  Stop.
make[1]: Leaving directory 'C:/GitHub Study/caravel_user_project'
Makefile:40: recipe for target 'setup' failed
make: *** [setup] Error 2

![image](https://github.com/efabless/caravel_user_project/assets/130479288/65845bbb-27e8-4fa2-aeb1-841b61e510f4)

**Comment by vijayank88** [comment]
Is WSL installed?
Or better use VM and load Ubuntu latest and  try.

**Comment by nicedayd** [comment]
Thank you for reponse! My laptop does not install WSL and I run it in Windows10 system. Do you have instruction that operater Caravel use project with WSL. My working laptop not allow to install VM but I will try to install WSL. Thanks. 


















At 2023-06-01 19:37:36, "vijayan" ***@***.***> wrote:

Is WSL installed?
Or better use VM and load Ubuntu latest and try.

**Reply by nicedayd** [reply]
Hello,
I installed the WSL and follow the tools installation manual of read but it reported error. Could you suggest how to fix it?

![image](https://github.com/efabless/caravel_user_project/assets/130479288/63dea352-10e5-4491-939e-602aaba992cb)

**Reply by adrienluitot** [reply]
It is not supposed to be done that way. 
Just do `sudo apt install make` (without the "setup").

`make` is the command to run a Makefile, `setup` is just a recipe/option of the project's Makefile.
